### PR TITLE
improve(DQ):error msgs

### PR DESCRIPTION
**Pull request by Irenez753**

### CODE REVIEW

File: src/main/scala/com/yotpo/metorikku/metric/stepActions/dataQuality/ValidationRunner.scala [hunk 1]
```diff
@@ -57,15 +57,20 @@ case class ValidationRunner() {
...
 
     logByLevel(verificationResult.status, validationCheckFailedMsg)
     checkResult.constraintResults.foreach { constraintResult =>
       constraintResult.metric match {
         case Some(metric: DoubleMetric) =>
           metric.value match {
             case Success(value) =>
-              logByLevel(verificationResult.status, doubleMetricConstrainFailedMsg.format((value * 100), metric.name))
+              metric.entity.toString match {
+                case "Column" => logByLevel(verificationResult.status, doubleMetricColumnConstrainFailedMsg.format((100 -(value * 100)), metric.name))
```
Comment (Doe-Ed) [inline src/main/scala/com/yotpo/metorikku/metric/stepActions/dataQuality/ValidationRunner.scala:70]: space after -

File: src/main/scala/com/yotpo/metorikku/metric/stepActions/dataQuality/ValidationRunner.scala [hunk 2]
```diff
@@ -57,15 +57,20 @@ case class ValidationRunner() {
...
     checkResult.constraintResults.foreach { constraintResult =>
       constraintResult.metric match {
         case Some(metric: DoubleMetric) =>
           metric.value match {
             case Success(value) =>
-              logByLevel(verificationResult.status, doubleMetricConstrainFailedMsg.format((value * 100), metric.name))
+              metric.entity.toString match {
+                case "Column" => logByLevel(verificationResult.status, doubleMetricColumnConstrainFailedMsg.format((100 - (value * 100)), metric.name))
+                case "Dataset" => logByLevel(verificationResult.status, doubleMetricDataSetConstrainFailedMsg.format(value, metric.name))
+                case _ =>
```
Comment (Doe-Ed) [inline src/main/scala/com/yotpo/metorikku/metric/stepActions/dataQuality/ValidationRunner.scala:72]: Is there a way to get here? Do we want to print/log error if we do?

File: examples/dq/movies_metric.yaml [hunk 3]
```diff
@@ -12,8 +12,9 @@ steps:
   dq:
     level: warn
     checks:
-      - isComplete:
```
Comment (Doe-Ed) [inline examples/dq/movies_metric.yaml:15]: Why remove?

File: examples/dq/movies_metric.yaml [hunk 4]
```diff
@@ -12,8 +12,9 @@ steps:
...
-          column: movieid
+      - hasSize:
+          size: 1000000000
+          operator: '>'
```
Comment (Doe-Ed) [inline examples/dq/movies_metric.yaml:17]: Why add this?

File: src/main/scala/com/yotpo/metorikku/metric/stepActions/dataQuality/ValidationRunner.scala [hunk 5]
```diff
@@ -57,15 +57,20 @@ case class ValidationRunner() {
...
     checkResult.constraintResults.foreach { constraintResult =>
       constraintResult.metric match {
         case Some(metric: DoubleMetric) =>
           metric.value match {
             case Success(value) =>
-              logByLevel(verificationResult.status, doubleMetricConstrainFailedMsg.format((value * 100), metric.name))
+              metric.entity.toString match {
+                case "Column" => logByLevel(verificationResult.status, doubleMetricColumnConstrainFailedMsg.format((100 - (value * 100)), metric.name))
+                case "Dataset" => logByLevel(verificationResult.status, doubleMetricDataSetConstrainFailedMsg.format(value, metric.name))
+                case "Mutlicolumn" => logByLevel(verificationResult.status, doubleMetricColumnConstrainFailedMsg.format((100 - (value * 100)), metric.name))
```
Comment (Irenez753) [inline src/main/scala/com/yotpo/metorikku/metric/stepActions/dataQuality/ValidationRunner.scala:72]:
```
object Entity extends Enumeration {
  val Dataset, Column, Mutlicolumn = Value
}
```
no other option

**Review by Doe-Ed — APPROVED** [review]

**Pull request merged**